pyproject.toml

```diff
@@ -144,6 +144,10 @@ follow_untyped_imports = true
 module = "tests.*"
 disallow_untyped_decorators = false # mypy gets confused by pytest decorators
 
+[[tool.mypy.overrides]]
+module = ["pytest", "mktestdocs", "pytest_lazy_fixtures", "syrupy.assertion"]
```

Comment:
We want to ignore only certain packages.

Reply:
Is the pyproject.toml file used when running mypy in the pre-commit hook through the run-mypy.sh script?

Reply:
Ok cool! That's the same command I use so if it picks up the pyproject.toml with that command, works for me 👍